ngrinder-frontend/src/js/main.js

```diff
@@ -12,6 +12,7 @@ import vuescroll from 'vuescroll';
 import numeral from 'numeral';
 import store from 'store/vuex-store.js';
 import VueRouterReferer from 'vue-router-referer.js';
+import VueClipboard from 'vue-clipboard2';
```

Comment:
module is already included.

https://github.com/naver/ngrinder/blob/develop/ngrinder-frontend/package.json#L52